### DISCUSSION TITLE
Maplibregl not found in leafmap

### DISCUSSION THREAD

**Discussion by DeutschZaw** [giswqs/geog-312#83] [Q&A] [answered]
At the lecture of "Intro to GIS Programming | Week 1: Introduction to Conda, Mamba, VS Code, and Git",
I have already installed correctly the leafmap but when I run the "overture.ipynb" (downloaded from leafmap website) in Jupyterlab, there was an error: it says:

"---------------------------------------------------------------------------
ModuleNotFoundError Traceback (most recent call last)
Cell In[2], line 1
----> 1 import leafmap.maplibregl as leafmap

File [~\anaconda3\envs\geo\Lib\site-packages\leafmap\maplibregl.py:10](http://localhost:8888/lab/tree/~/anaconda3/envs/geo/Lib/site-packages/leafmap/maplibregl.py#line=9)
8 import ipyvuetify as v
9 from box import Box
---> 10 from maplibre.basemaps import background
11 from maplibre.basemaps import construct_carto_basemap_url
12 from maplibre.ipywidget import MapWidget

ModuleNotFoundError: No module named 'maplibre'
"
I have attached the screenshot of this error.
![error](https://github.com/user-attachments/assets/9d0fbe7c-1939-4515-b3ad-53c245f0ebea)

**Comment by giswqs** [comment, marked as answer]
See https://github.com/giswqs/geog-312/issues/82#issuecomment-2385956021
